mailet/crypto/src/main/java/org/apache/james/transport/mailets/AbstractSign.java

```diff
@@ -294,14 +294,11 @@ protected void initKeyHolder() throws Exception {
         }
         
         if (isDebug()) {
-            StringBuilder logBuffer =
-            new StringBuilder(1024)
-            .append("KeyStore related parameters:")
-            .append("  keyStoreFileName=").append(keyStoreFileName)
-            .append(", keyStoreType=").append(keyStoreType)
-            .append(", keyAlias=").append(keyAlias)
-            .append(" ");
-            LOGGER.debug(logBuffer.toString());
+            LOGGER.debug("KeyStore related parameters:"
```

Comment:
Same here you can remove this now useless string concatenation

Reply:
Done